File: app/src/main/java/net/sarasarasa/basemvp/base/IMvpPresenter.java
package net.sarasarasa.basemvp.base;

/**
 * 封装Presenter层共有的一些显示方法
 */
public interface IMvpPresenter<V extends IMvpView> {

    void attachView(V view);

    void detachView(boolean retainInstance);
}

File: kotlin/src/main/java/net/sarasarasa/basemvpkotlin/base/IMvpPresenter.kt
package net.sarasarasa.basemvpkotlin.base

/**
 * 封装Presenter层共有的一些显示方法
 */
interface IMvpPresenter<in V : IMvpView> {

    fun attachView(view: V)

    fun detachView(retainInstance: Boolean)
}
